### ISSUE TITLE
Add a shortcut for a CloudWatch alarm triggering a Lambda Function

### ISSUE THREAD

**Issue by mcwhittemore** [mapbox/cloudfriend#106]
Getting a Lambda Function to be triggered by a CloudWatch Alarm requires one to create a SNS Topic and Lambda permission and wire them up correctly to the Alarm and Lambda function. These are boilerplate and feel like something we could shortcut so as to simplify the process of figuring this out. That said, I'm not sure what sort of barriers we want on the shortcuts we offer here. Any thoughts on if this makes the mark?

**Comment by rclark** [comment]
I think this'd be meaningful. AWS doesn't have a direct alarm --> lambda pathway yet?

**Comment by mcwhittemore** [comment]
Not that I have found and the example doc that hitting the top of google for me shows how to set this up via PowerShell which is pretty hard to parse. https://aws.amazon.com/blogs/developer/send-real-time-amazon-cloudwatch-alarm-notifications-to-amazon-chime/

**Comment by rclark** [comment]
[CloudWatch API reference docs](https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_PutMetricAlarm.html) confirm that you can only use certain services as alarm actions, and lambda isn't included. So yeah, you need to stick an SNS topic in between the alarm and the Lambda.
